en/docs/troubleshooting/troubleshooting.md

```diff
@@ -202,21 +218,114 @@ If you need to override the current Microgateway project, run the <code>
-| `                  900909                 ` | The subscription to the API is inactive.                                                                                                                                                                           |
-| `                  900910                 ` | The access token does not allow you to access the requested resource.                                                                                                                                              |
-| `                  900803                 ` | Application level throttled out.                                                                                                                                                                                   |
-| `                  900804                 ` | Subscription level throttled out.                                                                                                                                                                                  |
-| `                  900808                 ` | An internal error occurred in the Microgateway.                                                                                                                                                                    |
-| `                  900809                 ` | An internal error occurred in the Microgateway, since a subscription or application throttle policy is not deployed. This might be due to adding a throttle policy to API-M and not regenerating the Microgateway. |
+<table>
+<thead>
+<tr class="header">
+<th style="width: 5%">Error code</th>
+<th style="width: 40%">Error message</th>
+<th style="width: 55%">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td><code>900901</code></td>
+<td>Sandbox key offered to the API with no sandbox endpoint</td>
+<td>The production/sandbox key offered by the requested endpoint is not specified.</td>
+</tr>
+<tr>
+<td><code>900900</code></td>
+<td>Unclassified authentication failure</td>
+<td>An unspecified API-M authentication related error.</td>
+</tr>
+<tr>
+<td><code>900901</code></td>
+<td>Invalid credentials</td>
+<td>Invalid authentication information provided.</td>
+</tr>
+<tr>
+<td><code>900902</code></td>
+<td>Missing credentials</td>
+<td>No authentication information provided.</td>
+</tr>
+<tr>
+<td><code>900906</code></td>
+<td>No matching resource found in the API for the given request</td>
+<td>A resource with the name in the request cannot be found in the API.</td>
+</tr>
+<tr>
+<td><code>900907</code></td>
+<td>The requested API is temporarily blocked</td>
+<td>Happens when the API user is blocked from API-M.</td>
+</tr>
+<tr>
+<td><code>900908</code></td>
+<td>Resource forbidden</td>
+<td>The user invoking the API has not been granted access to the required resource.</td>
```

Comment:
The application which is used to get the authentication token does not have a valid subscription to the API